pkg/kotsadm/objects/license_objects.go

```diff
@@ -6,7 +6,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func LicenseSecret(namespace string, appSlug string, license string) *corev1.Secret {
+func LicenseSecret(namespace string, appSlug string, isAirgap bool, license string) *corev1.Secret {
+	annotations := map[string]string{
+		"kots.io/airgap": "false",
```

Comment:
this logic can be replaced with the follow (and then be part of the secret definition instead)
```
annotations := map[string]string{
  "kots.io/airgap": fmt.Sprintf("%t", isAirgap),
}
```

Reply:
looks neat, but the annotations seems to be map[string]string- https://pkg.go.dev/k8s.io/apimachinery@v0.20.4/pkg/apis/meta/v1#ObjectMeta.Annotations

Reply:
the sprintf will convert it to a string

Reply:
yes indeed:) https://play.golang.org/p/d0sV-jkzPXn fixed now